sdks/python/setup.py

```diff
@@ -102,6 +102,7 @@ def get_version():
   'google-apitools>=0.5.6,<1.0.0',
   'proto-google-cloud-datastore-v1==0.90.0',
   'googledatastore==7.0.0',
+  'google-cloud-bigquery>=0.22.1,<1.0.0',
```

Comment:
@markflyhigh can you clarify about the use of the new library vs existing client ?